### PR TITLE
#5483 Handling error on wrong  argument. Added missing language suppo…

### PR DESCRIPTION
**Pull request by michalgce**
…rt for imports.

### CODE REVIEW

**Comment by scottdraves** [comment]
please bring up to date with master

**Comment by scottdraves** [comment]
failing with:

> /home/ubuntu/jenkins/workspace/BeakerX PR/kernel/clojure/src/main/java/com/twosigma/beakerx/clojure/kernel/Clojure.java:61: error: method does not override or implement a method from a supertype

**Comment by scottdraves** [comment]
conflicts and test failure...

File: build.gradle [hunk 1]
```diff
@@ -13,27 +13,29 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-import org.apache.tools.ant.taskdefs.condition.Os
-
-
 task wrapper(type: Wrapper) {
-  gradleVersion = '3.5'
+    gradleVersion = '3.5'
```
Comment (scottdraves) [inline build.gradle]: please don't change the indentation.

**Comment by scottdraves** [comment]
i am still not getting a decent error msg...

**Comment by michalgce** [comment]
@scottdraves 
image

**Pull request merged**